packages/flutter/test/cupertino/checkbox_test.dart

```diff
@@ -1,6 +1,7 @@
 // Copyright 2014 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+import 'dart:ui';
```

Comment:
I believe that a blank line here will make the analyzer happy:

```suggestion

import 'dart:ui';
```